src/components/LuxUniversityFooter.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <component :is="type" :class="['lux-university-footer', theme]">
+    <lux-wrapper class="lux-footer-content" :maxWidth="maxWidth">
+      <div class="lux-footer-bottom bottom-layout">
+        <div class="lux-university-links">
+          <div class="policy-links">
+            <LuxHyperlink
+              href="https://library.princeton.edu/about/policies/copyright-and-permissions-policies"
+            >
+              Copyright Policy</LuxHyperlink
+            >
+            |
+            <LuxHyperlink href="https://www.princeton.edu/privacy-notice">
+              Privacy Notice
+            </LuxHyperlink>
+          </div>
+          <LuxHyperlink href="https://accessibility.princeton.edu/help"
+            >Accessibility Help</LuxHyperlink
+          >
+        </div>
+        <div class="lux-university-links center-panel">
+          <lux-university-copyright type="div" :theme="value(theme)" />
+        </div>
+        <div class="lux-university-links pu-logo-white">
+          <a href="https://www.princeton.edu"
+            ><lux-logo-university-white width="200px" height="72px" type="div"
+          /></a>
+        </div>
+      </div>
+    </lux-wrapper>
+  </component>
+</template>
+
+<script>
+import LuxUniversityCopyright from "./_LuxUniversityCopyright.vue"
+import LuxLogoUniversityWhite from "./logos/LuxLogoUniversityWhite.vue"
+import LuxUniversityPrivacyNotice from "./_LuxUniversityPrivacyNotice.vue"
+import LuxWrapper from "./LuxWrapper.vue"
+
+/**
+ * UniversityFooter is the preferred Footer styling/behavior for PUL websites.
```

Comment:
Could you add a bit more documentation about the difference between `LuxUniversityFooter` and `LuxLibraryFooter`, and when a site might choose to use just the University one vs. the whole Library one?

Reply:
Good idea. I took a pass at this.